hw/ip/aes/dv/aes_base_sim_cfg.hjson

```diff
@@ -78,54 +90,54 @@
   // List of test specifications.
   tests: [
     {
-      name: aes_wake_up
+      name: "{variant}_wake_up"
```

Comment:
should we keep the original names, otherwise, the common test are named `aes_*` while the tests defined here are called `aes_mask/unmask_*`?
Since we can apply the `mask/unmask` to common tests, I changed `{variant}` to `{name}` for sram_ctrl, which makes test names more consistent.
https://reports.opentitan.org/hw/ip/sram_ctrl_main/dv/latest/results.html

Reply:
Sounds good, I guess instead of using names, I can just revert back to original text right?